plugins/tech-insights-backend-module-jsonfc/package.json

```diff
@@ -46,8 +46,7 @@
     "winston": "^3.2.1"
   },
   "devDependencies": {
-    "@backstage/cli": "^0.16.0",
-    "@types/node-cron": "^3.0.1"
+    "@backstage/cli": "^0.16.0"
```

Comment:
now this actually also needs a changeset!

Reply:
oh fair since we ripped out node-cron completely. Added!